data.json

```diff
@@ -127,4 +127,12 @@
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "tags": ["stickers"]
   }
+  {
+    "name": "Aras",
+    "difficulty": "medium",
+    "description": "Submit an improvment to any <a href="http://community.aras.com/en/projects/">Aras Community Project</a> in October 2018 to earn stickers! Submit five, earn a limited edition T-shirt!",
```

Comment:
Double quotes need to be escaped within a string 😄😉

Reply:
Use single quotes instead!

Reply:
@ncallan2019 Please use @plibither8's solution instead